css/src/components/verticalMenu.css

```diff
@@ -0,0 +1,96 @@
+
+.VerticalMenu {
+  height: 100%;
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.VerticalMenu--expanded {
+  width: var(--spacing-9);
+}
+
+.VerticalMenu-section {
+  text-transform: uppercase;
+  padding-left: var(--spacing-2);
+  padding-top: 14px;
+  padding-bottom: 6px;
```

Comment:
Please use pre-defined spacing variables which are close to the corresponding values.